desktop/assets/js/jquery.desktop.x.js

```diff
@@ -659,6 +660,23 @@ JQDX.bindDocumentEventHandlers = function bindDocumentEventHandlers () {
     $(this).closest('tr').addClass('active');
   });
 
+  d.on('click', '#clock', function () {
+    const format = desktop.app.localstorage.get('clock');
+    // clear before we switch to a new format
+
+    if (desktop.clockTimeout) {
+      clearTimeout(desktop.clockTimeout);
+    }
+
+    // switch clock format depending on what is set
+    if (format && format === 'dd-mm-yyyy') {
+      desktop.app.localstorage.set('clock', 'mm-dd-yyyy');
```

Comment:
Instead of calling `localstorage.set`, we should be using `desktop.set`, same for `.get()`

This is to ensure future proof API for multiple storage data providers.